File: beginner/1060.cpp
#include <stdio.h>
 
int main() {
	float num[6];
	int count = 0;
	
    for(int i = 0; i < 6; i++){
    	scanf("%f", &num[i]);
    	if(num[i] >= 0.0) {
			count++;
		}
    }
    
    printf("%d valores positivos\n", count);
    
    
    return 0;
}

File: C#/begginer/1060.cs
using System; 

class URI {

  static void Main(string[] args) { 
    int count = 0;
    for(int i = 0; i < 6; i++) {
      double input = double.Parse(Console.ReadLine());
      if (input > 0) count++;
    }
    Console.WriteLine($"{count} valores positivos");
  }

}
